docs/main/user-guides/local-engine-user-guide/local-pv-hostpath/hostpath-configuration.md

```diff
@@ -0,0 +1,87 @@
+---
+id: hostpath-configuration
+title: Configuration
+keywords:
+ - OpenEBS Local PV Hostpath
+ - Local PV Hostpath Configuration
+ - Configuration
+description: This section explains the configuration requirements to set up OpenEBS Local Persistent Volumes (PV) backed by Hostpath. 
+---
+
+# Configuration
```

Comment:
##

Reply:
I am guessing the question is for this change. The heading type was changed into a title.